demo/pytriton_service/client.py

```diff
@@ -0,0 +1,10 @@
+from pytriton.client import ModelClient
```

Comment:
Thank you very much for your contribution. Please format the code and write the usage in front of the script and in https://github.com/open-mmlab/mmdetection/blob/v3.0.0rc6/docs/zh_cn/user_guides/useful_tools.md

Reply:
Sure.